### DISCUSSION TITLE
Ability to Head Back to Search Results Page in One Click

### DISCUSSION THREAD

**Discussion by rohithdsouza** [community/community#21288] [Other Feature Feedback, Questions, & Ideas]
This is just a suggestion for `cs.github.com`.

Currently in `cs.github.com` , if i search for a term -> open a specific result -> browse through some other files in the repository, when i try to head back to the search results page, i would have to click 'back' multiple times (~the amount of files i've navigated through the current repo) just to get to the search result page. This is cumbersome and time consuming, especially if one would like to browse through multiple repo files quickly from the search result page.

Hence, adding a functionality to return to the search results page in one click would be extremely helpful.

**Comment by colinwm** [comment]
Good suggestion! Do you think it should be a separate button somewhere on the page? Or we could also just make the back button go directly back to the search results page, rather than taking you back through code navigations.

**Reply by rohithdsouza** [reply]
Thank you!
I believe it's best if there was a separate button that would just take you back to the search result page, currently there are no back UI buttons, hence adding a button on the page (like '<') to head back to the search results page, while using the browser back button to browse through previously opened files would address both use-cases imo!

**Comment by thuongtruong1009** [comment]
Hi @rohithdsouza 
I think that you can right-click and open URL pages that you like in the new tab. So, you completely control whatever you like. If you want back, you only close that tab.

**Reply by rohithdsouza** [reply]
Thank you! that's definitely one way to handle heading back to the search result page faster!
